README.md

```diff
@@ -366,6 +368,21 @@ Note that:
 
 - `mainnet` requires an additional `token` parameter to submit transactions
 - `starknet_addDeployTransaction` and `starknet_addDeclareTransaction` allow an optional `abi` field
+
+## Monitoring API
+
+Pathfinder has a monitoring API which can be enabled with the `--monitor-address` configuration option.
+
+### Health
+
+`/health` provides a method to check the health status of your `pathfinder` node. It returns `OK(200)` if the node is healthy.
```

Comment:
Could add something about this aiming to be k8s friendlier. With `OK(200)` do you mean it's an empty response but we set 200 OK as the status or that it will in fact have `OK(200)` in it's body with 200 OK status?

Reply:
The former. Was unclear on the conventional way of writing this.